crypto_sign/dilithium2/clean/ntt.c

```diff
@@ -113,25 +114,25 @@ void PQCLEAN_DILITHIUM2_CLEAN_ntt(uint32_t p[N]) {
 *
 * Arguments:   - uint32_t p[N]: input/output coefficient array
 **************************************************/
-void PQCLEAN_DILITHIUM2_CLEAN_invntt_frominvmont(uint32_t p[N]) {
-    unsigned int start, len, j, k;
+void PQCLEAN_DILITHIUM2_CLEAN_invntt_frominvmont(uint32_t *p) {
```

Comment:
What's the reason for changing these?

Reply:
This was still `uint32_t *p` in the upstream code.

Reply:
Ah, no, weird, this was present in the upstream code. Not sure how this got changed.